features/facebookImport/src/components/importExplanationExpandable/importExplanationExpandable.jsx

```diff
@@ -162,7 +164,13 @@ const ImportExplanationExpandable = ({
                     ) : selectedFileUrl ? (
                         <div className="file-info">
                             <h5>{i18n.t("import:import.chosen")}</h5>
-                            <p>{selectedFileUrl?.split("/").pop()}</p>
+                            <p>
+                                {selectedFileUrl
+                                    ?.split("/")
+                                    .pop()
+                                    .split(escapedSlash)
```

Comment:
Maybe URI decode it first?

Reply:
Doesnt work sadly, everyones paths seem to be different. Some just pure weirdness not even having the file name in it. So @fhd and I decided to go back to ExternalFile Object here

Reply:
But is it because it's not roundtripping correctly when we put it through the pod storage and all that? Anyway, as I said, if it works, it works.

Reply:
No idea, it's only android so it think the paths are just different from system to system

Reply:
Works for me but not for other people, which means it doesnt work

Reply:
@JJ It is round tripping correctly, the problem is that the URL we actually get from Android's native file selector _is not what we expect_. We, or well, Richard here (:P), expected it to _contain the name of the selected file as the last path segment_. That is, a fair assumption. Instead, we get something different - and it's even different across different Android devices.

Therefore, I believe the best we can do is to treat it as a handle and don't try to make sense of it or get specific information out of it at all. In the native code, we _do_ get the actual file name, so it's easier to include it next to the URL.